### ISSUE TITLE
Provide binary signatures

### ISSUE THREAD

**Issue by robeden** [dome9/shiftleft#1]
Especially for a security tool, it's really not great to have an unsigned binary be an integral part of the process. At a minimum, published checksums would be good for binaries. 

Bonus points for signing the Mac binary with a real developer signature so Gatekeeper doesn't need to be disabled. 🙂

**Comment by ShivonQ** [comment]
Yeah it's sort of wild this is not signed guys.  Please sign this binary!
